zkvm-prover/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM rust:1.85
+
+RUN rustup toolchain install nightly-2025-02-14-x86_64-unknown-linux-gnu
+RUN rustup component add rust-src --toolchain nightly-2025-02-14-x86_64-unknown-linux-gnu
+
+WORKDIR /app
+RUN git clone --branch refactor/scroll_mono --single-branch --depth 1 https://github.com/scroll-tech/zkvm-prover.git zkvm-circuits
+
+ADD common/types-rs /app/common/types-rs
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)